low-level/files/opt/seagate/sspl/bin/sspl_reset.py

```diff
@@ -0,0 +1,174 @@
+import sys
+import errno
+import argparse
+import inspect
+import traceback
+import os
+import shutil
+import subprocess
+# Add the top level directories
+sys.path.insert(0, '/opt/seagate/cortx/sspl/low-level')
+
+from framework.base.sspl_constants import PRODUCT_FAMILY, file_store_config_path, enabled_products
+
+SSPL_CONFIGURED=f"/var/{PRODUCT_FAMILY}/sspl/sspl-configured"
+
+if os.path.exists(file_store_config_path):
```

Comment:
move this logic into the below parent class (in Cmd)

Reply:
Done.